android/lib/src/main/java/com/swmansion/gesturehandler/PinchGestureHandler.kt

```diff
@@ -59,12 +59,12 @@ class PinchGestureHandler : GestureHandler<PinchGestureHandler>() {
       spanSlop = configuration.scaledTouchSlop.toFloat()
       begin()
     }
-    scaleGestureDetector?.onTouchEvent(event)
-    var activePointers = event.pointerCount
-    if (event.actionMasked == MotionEvent.ACTION_POINTER_UP) {
-      activePointers -= 1
+  
+  if(event.actionMasked != MotionEvent.ACTION_POINTER_UP){
```

Comment:
This surely appears weird, but I am not sure If there's a better way to achieve this. This is done due to the below scenario.

User has 2 fingers and performing pinch -> One finger is lifted -> ACTION_POINTER_UP is fired -> event.pointerCount is still 2 -> scale detector sets the focal values to single finger -> the event received has new focus but the pointer count is still 2.

However, this again makes it consistent with iOS. iOS Pinch handler doesn't send events in case of a finger is lifted. Only when a single pointer is moved, it starts sending the event.

Reply:
One more way would be to override the numberOfPointers in PinchGestureHandler and reduce it by 1 when Action is Pointer up but then we'll have to make it `open` in GestureHandler 😓